hermes/serializers.py

```diff
@@ -138,59 +138,246 @@ class Meta(BaseTargetSerializer.Meta):
+        return ra_angle.deg
+
+    def validate_dec(self, value):
+        try:
+            float_dec = float(value)
+            if not math.isfinite(float_dec):
+                raise serializers.ValidationError(_("Dec value must be finite"))
+            dec_angle = Latitude(float_dec * units.deg)
+        except (ValueError, units.UnitsError, TypeError):
+            try:
+                dec_angle = Latitude(value, unit=units.hourangle)
+            except:
+                try:
+                    dec_angle = Latitude(value)
+                except:
+                    raise serializers.ValidationError(_("Must be in a format astropy understands"))
+        return dec_angle.deg
+
+class CommonDataSerializer(serializers.Serializer):
     target_name = serializers.CharField(required=True)
-    ra = serializers.CharField(required=True)
-    dec = serializers.CharField(required=True)
-    date = serializers.CharField(required=True)
-    date_format = serializers.CharField(required=False)
+    date_obs = serializers.CharField(required=False)
     telescope = serializers.CharField(required=False, default='', allow_blank=True)
     instrument = serializers.CharField(required=False, default='', allow_blank=True)
-    band = serializers.CharField(required=True)
-    brightness = serializers.FloatField(required=True)
-    nondetection = serializers.BooleanField(required=False, default=False)
-    brightness_error = serializers.FloatField(required=False)
-    brightness_unit = serializers.ChoiceField(required=False, choices=["AB mag", "Vega mag", "mJy", "erg / s / cm² / Å"])
 
     def validate(self, data):
         validated_data = super().validate(data)
-        validate_date(validated_data['date'], validated_data.get('date_format'))
+        validate_date(validated_data['date_obs'])
         if not (validated_data.get('instrument') or validated_data.get('telescope')):
             error_msg = _("Must have at least one of telescope or instrument set")
             raise serializers.ValidationError({'telescope': error_msg, 'instrument': error_msg})
         return validated_data
 
+
+class PhotometryDataSerializer(CommonDataSerializer):
+    bandpass = serializers.CharField(required=True)
+    brightness = serializers.FloatField(required=False)
+    brightness_error = serializers.FloatField(required=False)
+    brightness_unit = serializers.ChoiceField(required=False, default="AB mag", choices=["AB mag", "Vega mag", "mJy", "erg / s / cm² / Å"])
+    new_discovery = serializers.BooleanField(default=False, required=False)
+    exposure_time = serializers.FloatField(required=False)
+    observer = serializers.CharField(required=False)
+    comments = serializers.CharField(required=False)
+    limiting_brightness = serializers.FloatField(required=False)
+    limiting_brightness_unit = serializers.ChoiceField(required=False, default="AB mag", choices=["AB mag", "Vega mag", "mJy", "erg / s / cm² / Å"])
```

Comment:
Should we include a `limiting_brightness_error`?
If we have both a `limiting_brightness_unit` and a `brightness_unit` it seems like we should have an error too.

Reply:
I'm happy to add it. If you want it, then that's a strong argument others might want it too.

Reply:
I don't think it would hurt anything to have it available for people.